src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,26 @@
  */
 trait VarDumperTestTrait
 {
+    /** @internal */
+    private static $casters = [];
+    /** @internal */
+    private static $flags;
+    /** @internal */
+    private static $varCloner;
+
+    protected static function setUpVarDumper(array $casters, int $flags = null): void
+    {
+        self::$casters = $casters;
+        self::$flags = $flags;
+        self::$varCloner = null;
+    }
+
+    protected static function tearDownVarDumper(): void
```

Comment:
what about adding the `@after` annotation?

Reply:
That won't work if someone is calling `setUpVarDumper` in `setUpBeforeClass` once for the test class. But there is probably no overhead to recommend doing it always in `setUp` instead so a fresh `VarCloner` instance is used for each test case instead of one for the whole test class. Then the static API isn't even required.
As you wish.

Reply:
The less state managed by traits the better. We can add both `@after` and `@afterClass`